src/meta/Intersection.ts

```diff
@@ -0,0 +1,62 @@
+import global from '@dojo/core/global';
+import Map from '@dojo/shim/Map';
+import WeakMap from '@dojo/shim/WeakMap';
+import MetaBase from '../meta/Base';
+
+import 'intersection-observer';
+
+export interface IntersectionOptions {
+	root?: string;
+}
+
+export default class Intersection extends MetaBase {
+	private _listeners = new WeakMap<Element, IntersectionObserverEntry>();
+	private _rootObservers = new Map<string, IntersectionObserver>();
+
+	private _onIntersect(entries: IntersectionObserverEntry[], observer: IntersectionObserver) {
+		entries.forEach(entry => {
```

Comment:
the style guide says should use brackets for arrow functions `forEach((entry) => ...`